lib/rubocop/cop/rspec/subject_stub.rb

```diff
@@ -74,71 +75,59 @@ class SubjectStub < Cop
         PATTERN
 
         def on_block(node)
+          # process only describe/context/... example groups
```

Comment:
Usually, the code is pretty self-explanatory.
If you wish to clarify what's going on where, it's ok to put comments here for the reviewers.

Reply:
@pirj So would you recommend to remove all the added comments? (I updated the PR a little to pass Rubocop checks)

Reply:
The issue with comments is that the code seems obvious when you spends a lot of time thinking about it.

Okey, I've removed all the comments.